src/TriggerBinding/SqlTriggerBinding.cs

```diff
@@ -107,7 +107,7 @@ private string GetUserFunctionId()
             string websiteName = SqlBindingUtilities.GetWebSiteName(this._configuration);
 
             var methodInfo = (MethodInfo)this._parameter.Member;
-            string functionName = $"{methodInfo.DeclaringType.FullName}.{methodInfo.Name}";
+            string functionName = $"{methodInfo.DeclaringType.Name}.{methodInfo.Name}";
```

Comment:
Based on the discussion in other comments, shouldn't this be just `methodInfo.DeclaringType.Name`. Why're we appending `methodInfo.Name` as well now? I think the ScaleController wouldn't have the info around `methodInfo.Name`

Reply:
`methodInfo.DeclaringType.Name` is the className of the function and not the function name as I assumed, so if a class has two functions against the same table then that would only trigger one function (one of our tests was set up that way and that's how I caught it). @AmeyaRele Could you confirm if you can access method name on ScaleController end?

Reply:
@vivekjilla I just synced with @AmeyaRele  and updated the code to read the function name from `FunctionName` attribute. Will merge mu changes in once the tests pass and get on with the releasing the preview package tomorrow.

Reply:
Thank you!